rocky/account/models.py

```diff
@@ -79,3 +84,14 @@ class KATUser(AbstractBaseUser, PermissionsMixin):
 
     def get_full_name(self):
         return self.full_name
+
+    @cached_property
+    def organizations(self) -> List[Organization]:
+        """Lists all organizations a user is a member of, excluding organizations to which access is blocked.
+
+        Superusers are considered to be members of all organizations.
+        """
+        if self.is_superuser:
+            return list(Organization.objects.all())
+        members = self.members.exclude(status=OrganizationMember.STATUSES.BLOCKED).select_related("organization")
+        return [member.organization for member in members]
```

Comment:
Regarding this cached property: to me this implementation balances performance and legibility.